rapids/verify_rapids_dask.py

```diff
@@ -1,39 +1,35 @@
-import cudf
```

Comment:
Cuda accelerated data frames is an important component.  I don't think we want to disable tests for it.